.editorconfig

```diff
@@ -20,24 +17,13 @@ csharp_style_expression_bodied_constructors = false:silent
 csharp_style_expression_bodied_operators = false:silent
 csharp_indent_labels = one_less_than_current
 
-[*.sln]
-indent_style = tab
-
 # Xml project files
 [*.{config,csproj,njsproj,targets,vcxitems,vcxproj,vcxproj.filters}]
 indent_size = 2
-end_of_line = crlf
-insert_final_newline = false
 
 # XML config files
 [*.{msbuild,props,targets,ruleset,config,nuspec}]
 indent_size = 2
-end_of_line = crlf
-insert_final_newline = false
```

Comment:
I'd say we should leave insert_final_newline = false in. These are specific VS formats. VS never adds final new lines and removes them when found. It is safe and reasonable to leave this setting in the editorconfig, so other code editors won't accidentially insert final new lines when it is their default.